sdk/src/metrics/meter_context.cc

```diff
@@ -79,6 +80,26 @@ void MeterContext::AddMeter(std::shared_ptr<Meter> meter)
   meters_.push_back(meter);
 }
 
+void MeterContext::RemoveMeter(nostd::string_view name,
+                               nostd::string_view version,
+                               nostd::string_view schema_url)
+{
+  std::lock_guard<opentelemetry::common::SpinLockMutex> guard(meter_lock_);
+
+  std::vector<std::shared_ptr<Meter>> filtered_meters;
+
+  for (auto &meter : meters_)
+  {
+    auto scope = meter->GetInstrumentationScope();
+    if (!scope->equal(name, version, schema_url))
+    {
+      filtered_meters.push_back(meter);
+    }
+  }
+
+  meters_.swap(filtered_meters);
```

Comment:
nit - should we log a warning if the meter to be removed is not found?

Reply:
Thanks.

Printed an INFO message when removing a meter instead, this is better and provides more debug help if needed.